frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx

```diff
@@ -31,9 +33,21 @@ const DataFormat: FC<Props> = ({
         onChange={handleChange}
         selectedValue={format}
       >
-        <Radio label=".h5ad (AnnData v0.7)" value={DATASET_ASSET_FORMAT.H5AD} />
-        <Radio label=".loom" value={DATASET_ASSET_FORMAT.LOOM} />
-        <Radio label=".rds (Seurat v3)" value={DATASET_ASSET_FORMAT.RDS} />
+        <Radio
```

Comment:
Update Download Dataset Modal to disable unavailable formats, either because the format failed or is still converting